linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_unlink/rule.yml

```diff
@@ -71,3 +90,9 @@ template:
           - renameat
           - unlink
           - unlinkat
+    {{% else %}}
+    name: audit_rules_syscal_events
```

Comment:
It should be syscall (with 2 "l"s).

Reply:
Hello @jan-cerny and thank you very much for your proposal. I have done some corrections and this commit covers them https://github.com/ComplianceAsCode/content/pull/7529/commits/b8e1c44b3cc422d20b362097d72c49879049a2d8